homeassistant/components/mobile_app/binary_sensor.py

```diff
@@ -49,9 +49,19 @@ def handle_sensor_registration(webhook_id, data):
 
         async_add_entities([MobileAppBinarySensor(data, device, config_entry)])
 
-    async_dispatcher_connect(hass,
-                             '{}_{}_register'.format(DOMAIN, ENTITY_TYPE),
-                             partial(handle_sensor_registration, webhook_id))
+    handles = hass.data[DOMAIN][DATA_SIGNAL_HANDLES][ENTITY_TYPE]
+    handles[webhook_id] = \
+        async_dispatcher_connect(hass,
+                                 '{}_{}_register'.format(DOMAIN, ENTITY_TYPE),
+                                 partial(handle_sensor_registration,
+                                         webhook_id))
+
+
+async def async_unload_entry(hass, entry):
```

Comment:
There is no such thing that is being called for platforms. The Entity Component handles it.